assets/js/base/components/button/style.scss

```diff
@@ -1,7 +1,6 @@
 .wc-block-components-button:not(.is-link) {
 	align-items: center;
 	display: inline-flex;
-	min-height: 3em;
```

Comment:
This seems to cause the button not to look good anymore in Storefront:

![imatge](https://user-images.githubusercontent.com/3616980/234851060-a209a51a-f070-4797-8b8d-c23bbde46db5.png)

I wonder if one solution would be to move it under `body:not(.woocommerce-block-theme-has-button-styles) .wc-block-components-button:not(.is-link)`? :thinking:

Reply:
Can you confirm this is on the cart/basket page in Storefront? Because it looks as expected for me 🤔 

![Screenshot 2023-04-27 at 13 43 24](https://user-images.githubusercontent.com/8639742/234865235-a70f19b7-4c03-411c-9814-2202101b9242.png)

Reply:
Good spot and suggestion, when I checked this for Cart I didn't realise I had the shortcode version enabled. Should be fixed in https://github.com/woocommerce/woocommerce-blocks/pull/9260/commits/d9c312017941eea1c9d35d189b8ff1f19d42405b